### DISCUSSION TITLE
VRP: consider 3D dimension of the loads

### DISCUSSION THREAD

**Discussion by Westlife1002** [google/or-tools#2855] [Routing (and legacy CP) questions] [answered]
cross-posted: https://groups.google.com/g/or-tools-discuss/c/qPZzcm5Q_H8 

I am wondering if ortools has the capability of considering the 3D dimension of the individual loads.  

Most other VRP models does not take into account this dimension, likely due to the complexity or computing efficiency. However, this is an extremely import dimension in reality to at least ensure the loads consolidated can be actually loaded in a vehicle. 

Since every loads can be different sizes in terms of length/wideth/height, wich will affect the utilization of vehicle space. To ensure the VRP result is executable, we often make a discount of the vehicle space capacity, like 60-70%. However, it's very hard to find the best balance point to ensure space is not wasted too much while more loads can be loaded. 

Are there any good strategy to deal with this issue? For now, we can forget about the weight of the loads for the sake of simplicity, just the volume (cubic meters) of loads.

**Comment by lperron** [comment, marked as answer]
Answered on the mailist list. Let's continue the discussion there.
